var/spack/repos/builtin/packages/tbl2asn/package.py

```diff
@@ -13,13 +11,18 @@ class Tbl2asn(Package):
        sequence records for submission to GenBank."""
 
     homepage = "https://www.ncbi.nlm.nih.gov/genbank/tbl2asn2/"
+    maintainers = ['snehring']
 
-    version('2020-03-01', sha256='7cc1119d3cfcbbffdbd4ecf33cef8bbdd44fc5625c72976bee08b1157625377e')
+    version('2022-04-26', sha256='c76481700e196ebd98a83f4174e0146569db9d6fe5753ac18691e9836d5c6a75')
+    version('2020-03-01', sha256='7cc1119d3cfcbbffdbd4ecf33cef8bbdd44fc5625c72976bee08b1157625377e', deprecated=True)
```

Comment:
I am not getting that (old) hash, rather:

```suggestion
    version('2020-03-01', sha256='c76481700e196ebd98a83f4174e0146569db9d6fe5753ac18691e9836d5c6a75', deprecated=True)
```

Reply:
That seems to match the newly added version. Which make sense, because they only keep the one file available. The old version would likely match that existing hash if it were actually available from their website still. I'm only leaving it in because someone might have it in local mirror or something.